src/routes/console/project-[project]/settings/migrations/(import)/step1.svelte

```diff
@@ -24,15 +23,15 @@
         nhost: 'NHost'
     };
 
-    function connectGoogle() {
-        const redirect = new URL($page.url);
-        const target = new URL(
-            `${sdk.forProject.client.config.endpoint}/migrations/firebase/connect`
-        );
-        target.searchParams.set('redirect', redirect.toString());
-        target.searchParams.set('projectId', $page.params.project);
-        return target;
-    }
+    // function connectGoogle() {
+    //     const redirect = new URL($page.url);
+    //     const target = new URL(
+    //         `${sdk.forProject.client.config.endpoint}/migrations/firebase/connect`
+    //     );
+    //     target.searchParams.set('redirect', redirect.toString());
+    //     target.searchParams.set('projectId', $page.params.project);
+    //     return target;
+    // }
```

Comment:
Is this needed?

Reply:
This is for OAuth, which is working but hidden for now pending Google's approval, so I'd rather keep it here so we can easily add it back.